screens/NoFriends/index.tsx

```diff
@@ -0,0 +1,67 @@
+import PerlinNoise from "@/components/PerlinNoise"
+import { CustomButton } from "@/components/ui/Button"
+import CustomText from "@/components/ui/CustomText"
+import { theme } from "@/theme"
+import { Image, StyleSheet, View } from "react-native"
+import { useSharedValue } from "react-native-reanimated"
+
+export default function NoFriendsScreen() {
+  const isOn = useSharedValue(false)
+
```

Comment:
it's pretty much the same as `Home.tsx` we just need to render the extra stuff in home instead